demo/src/test.js

```diff
@@ -125,6 +125,19 @@ describe('Regression Tests', () => {
 
  });
 
+ it("should remove inline styles from element", () => {
+  const se = getEditor();
+  const content = node(
```

Comment:
This might be what is failing. Is there a ```node``` function defined? 

I know this would work for sure:

```js
const content = document.createElement('div');
content.innerHTML = '<div style="position: static"></div><div style="position: static"></div>';
```

Reply:
```
    const se = getEditor();
    const content = document.createElement("div");
    content.innerHTML =
      '<div style="position: static"></div><div style="position: static"></div>';

    const pruned = se.pruneInlineStyles(content);
    console.log(pruned);
```
it outputs:
`<div><div style="position: static"></div><div style="position: static"></div></div>`

Reply:
![image](https://user-images.githubusercontent.com/38187639/94887697-ac1ef600-047f-11eb-80ce-6c52efbd532b.png)
![image](https://user-images.githubusercontent.com/38187639/94887703-b2ad6d80-047f-11eb-8a06-0e778169a677.png)

Reply:
I totally forgot the purpose of this function 🤣 

It is supposed to set ```positon: static``` regardless of any of these inline styles:

* ```relative```
* ```fixed```
* ```absolute```
* ```sticky```

We should pass content that has elements with ```position``` set to all these different properties and insure that they are all set to ```static```.

**Sorry about that!** 😝

Reply:
@unitehenry I changed test conditions, check it out